src/clients/repositories.rb

```diff
@@ -323,15 +327,20 @@ def repoInfoRichText(name, raw_url, category)
         "\">&nbsp;&nbsp;&nbsp;"
       )
 
+      url = _("Unknown") if url == ""
       raw_url = _("Unknown") if raw_url == ""
 
-      url = Builtins.sformat(_("URL: %1"), raw_url)
+      url_string = Builtins.sformat(_("URL: %1"), url)
+      if url != raw_url
+        url_string += "<BR>"
+        url_string += Builtins.sformat(_("Raw URL: %1"), raw_url)
```

Comment:
NP: What about `_("Raw URL: %s") % raw_url` to avoid adding an obsolete builtin...?